decidim-core/app/views/decidim/endorsements/identities.html.erb

```diff
@@ -1,12 +1,18 @@
-<div class="reveal__header reveal__header--nomargin reveal__bg p-s">
-  <h6 class="heading6 m-none reveal__title"><%= t ".select_identity" %></h6>
+<div data-dialog-container>
```

Comment:
_"If something could go wrong, it will"_

This is a very bad timing. It results we included a commit which does not include such button (due to remote modals). Then I fixed, but you has tested before last deployed ¬¬

Look [now](https://decidim-redesign.populate.tools/processes/book-bless/f/9/posts/5) how it looks:

![imagen](https://user-images.githubusercontent.com/817526/204579989-443671c3-e12a-4c08-b825-ed79d7aed5ea.png)